packages/e2e-test-utils-playwright/src/editor/switch-editor-tool.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Clicks a block toolbar button.
+ *
+ * @param this
+ * @param label The text string of the button label.
+ */
+export async function switchEditorTool( this: Editor, label: string ) {
```

Comment:
Nit: Prefer using unions of string literals in types since we only have two options for now. Something like `'Design' | 'Write'`.

Reply:
I'm hesitant, someone could decide to test another language.

Reply:
They're only types so I guess they can always bypass that. We can also add `| string` to the end to allow arbitrary strings.